### DISCUSSION TITLE
Django Project "Abstract"

### DISCUSSION THREAD

**Discussion by pauloxnet** [django-commons/membership#89] [Ideas]
It will be useful to have an "Abstract=True" (to cite something related to Django) project in the organization that implements all the best practices (eg: pyproject.toml, requirements, Makefile/Justfile, GitHub actions, Readme, Issue or PR templates, Changelog, CoC, Contribution doc, ...).
All projects in the organization can refer to it to align (if they want) and be sure they are using the best practices, so that they can simply align with it, without reinventing the wheel every time or study to find a way to do things (eg. Pipelines, ...) and can concentrate on the code of their package.

**Comment by tim-schilling** [comment]
I think that was my plan with the playground repo originally. However, perhaps that's a bad name.

Regardless, we could do a better job of marketing that and adding more to it.

**Reply by ulgens** [reply]
@tim-schilling Do we have a doc or ongoing discussion about these best practices?

**Reply by tim-schilling** [reply]
Nope, not yet! Feel free to create discussions for specific best practice topics. Try to avoid lumping multiple together unless they do belong together.

If we need to reorg the discussion topics later a django commons admin can do that.

**Reply by ulgens** [reply]
@tim-schilling Here we go https://github.com/orgs/django-commons/discussions/94

**Reply by tim-schilling** [reply]
I've renamed the playground repo to `best-practices` https://github.com/django-commons/best-practices

**Reply by cunla** [reply]
There are implications on the terraform state - I think this should be done via terraform.. 
I will have a look this week

**Reply by tim-schilling** [reply]
Okie dokie. I'll revert

**Comment by Stormheg** [comment]
Possibly [cruft](https://cruft.github.io/cruft/) could be of interest here. It helps backport (best practice) boilerplate code from a cookiecutter project template.
